Lib/test/test_email/test_message.py

```diff
@@ -985,6 +985,30 @@ def test_get_body_malformed(self):
         # AttributeError: 'str' object has no attribute 'is_attachment'
         m.get_body()
 
+    def test_get_payload_strip_ex_space(self):
+        """test for gh-123742"""
```

Comment:
Docstrings in unit test methods clutter the test report when you run the test in verbose mode (`-v`). Instead, use a concise regular comment in the form `# gh-xxxxxx: test foo bar baz because problems with blah blah blah`

Moreover, explain also what the issue number is about. It saves future readers of the code from having to open GitHub, search for the GitHub issue, and try to grasp what the problem was. Always keep future **readers** of your code in mind.